### PR TITLE
DOC improve readme

### PR DESCRIPTION
**Pull request by adrinjalali**
Improving the README and adding a CONTRIBUTING guideline.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
+    pip install skops
 
-Demo a model using HuggingFace Spaces
-=====================================
+Bug Reports and Questions
+-------------------------
 
-Deploy a model using Seldon
-===========================
+Please as all your questions and report issues on this repostory's issue
```
Comment (BenjaminBossan) [inline README.rst:48]:
```suggestion
Please send all your questions and report issues on this repository's issue
```

**Comment by adrinjalali** [comment]
Ready for another review, and the README can be seen rendered as this: https://github.com/adrinjalali/skops/blob/readme/README.rst

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
+Contributing to skops
+=====================
+
+Please follow this workflow when contributing to skops:
```
Comment (BenjaminBossan) [inline CONTRIBUTING.rst:4]:
```suggestion
Please follow this workflow when contributing to skops:

```

rst requires an empty line, lovely.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- Create a new branch for your changes
+- Add your changes to the branch
+- Commit your changes
+- Push your branch to the remote repository
+- Create a pull request on GitHub
+
+Review Process
+--------------
+
+Don't hesitate to ping @skops-dev/mintainers in your issues and pull requests
```
Comment (BenjaminBossan) [inline CONTRIBUTING.rst:16]:
```suggestion
Don't hesitate to ping @skops-dev/maintainers in your issues and pull requests
```

Unless this was on purpose ;)

**Review by BenjaminBossan — APPROVED** [review]
LGTM, thanks, only 2 minor issues.

**Comment by adrinjalali** [comment]
@BenjaminBossan CI is green now :)

**Pull request merged**